openadapt/events.py

```diff
@@ -225,22 +265,30 @@ def get_merged_events(
         logger.debug(f"{len(merged_events)=}")
         return merged_events
 
-
     return merge_consecutive_action_events(
-        "mouse_move", events, is_target_event, get_merged_events,
+        "mouse_move",
+        events,
+        is_target_event,
+        get_merged_events,
     )
 
 
-def merge_consecutive_mouse_scroll_events(events):
-    """Merge consecutive mouse scroll events into a single scroll event"""
+def merge_consecutive_mouse_scroll_events(events: list) -> list:
```

Comment:
`list[ActionEvent]) -> list[ActionEvent]`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507